oneview_redfish_toolkit/api/scmb.py

```diff
@@ -61,14 +62,18 @@ def _scmb_key_path():
     return os.path.join(_scmb_base_dir(), SCMB_KEY_NAME)
 
 
-def check_cert_exist():
+def has_valid_certificates():
     try:
         _scmb_base_dir()
     except KeyError as error:
         logging.error("Invalid configuration for ssl cert. "
                       "Verify the [ssl] section in config file")
         raise error
 
+    return _has_scmb_certificate() and is_cert_working_with_scmb()
```

Comment:
Since **_has_scmb_certificate()** internally will call **_scmb_base_dir()**, maybe we could put just this line inside the "**try**" block.